test/schemas/eventSchemas.test.ts

```diff
@@ -402,23 +402,15 @@ describe('The "goalCompleted" payload schema', () => {
             'Invalid format at path \'/goalId\'.',
         ],
         [
-            {goalId: '123foo'},
-            'Invalid format at path \'/goalId\'.',
-        ],
-        [
-            {goalId: '_foo'},
-            'Invalid format at path \'/goalId\'.',
-        ],
-        [
-            {goalId: '-foo'},
+            {goalId: 'foo*bar'},
             'Invalid format at path \'/goalId\'.',
         ],
         [
             {goalId: ':foo'},
             'Invalid format at path \'/goalId\'.',
         ],
         [
-            {goalId: 'foo*bar'},
+            {goalId: 'foo:'},
```

Comment:
missing tests ending with `-` and `_`.  Also, pls add tests with consecutive `_`, `-` and `:`

Reply:
Sure!